marklogic-data-hub/src/test/java/com/marklogic/hub/HubTestBase.java

```diff
@@ -183,8 +184,10 @@ protected HubClient runAsDataHubOperator() {
     @Override
     protected HubClient runAsUser(String mlUsername, String mlPassword) {
         hubClient = null;
-
-        applyMlUsernameAndMlPassword(mlUsername, mlPassword);
+        Properties props = hubConfigInterceptor.getHubConfigObjectFactory().getGradleProperties();
+        props.setProperty("mlUsername", mlUsername);
```

Comment:
I think it would be better to construct a new Properties object and copy everything from "props" into it, then override mlUsername/mlPassword. That ensures we don't modify the set of Properties that were actually read from disk. 

Good catch though figuring this out.